internal/backend/azure/azure.go

```diff
@@ -207,25 +207,41 @@ func (be *Backend) Path() string {
 	return be.prefix
 }
 
+// useAccessTier determines whether to apply the configured access tier to a given file.
+// It returns true if the configured AccessTier is not "archive" and the file is a non-metadata pack file.
+// This ensures that:
+// 1. No files are ever stored in the Archive tier, regardless of the AccessTier setting.
+// 2. Only data files (non-metadata pack files) use the configured AccessTier when it's not "archive".
+func (be *Backend) useAccessTier(h backend.Handle) bool {
+	noArchiveClass := strings.ToLower(be.cfg.AccessTier) != "archive"
+	isDataFile := h.Type == backend.PackFile && !h.IsMetadata
+	return isDataFile && noArchiveClass
```

Comment:
Why is that logic different from the one used for the S3 backend? Metadata files must use an online tier, but any such tier would work.

Reply:
Fixe. I overlooked the `||` and used an `&&` by mistake.